src/pykeen/models/nbase.py

```diff
@@ -332,7 +333,8 @@ def __init__(
         interaction_kwargs: Optional[Mapping[str, Any]] = None,
         entity_representations: EmbeddingSpecificationHint = None,
         relation_representations: EmbeddingSpecificationHint = None,
-        loss: Optional[Loss] = None,
+        loss: HintOrType[Loss] = None,
+        loss_kwargs: Optional[Mapping[str, Any]] = None,
```

Comment:
we should add an abbrev for this into types 😄

Reply:
`OptionalKwargs`?

Reply:
Could also go into `class_resolver`

Reply:
just made new release with class resolver with it https://pypi.org/project/class-resolver/0.1.1/

Reply:
#719